XMLtoMD/GuidelineXmlToMD/GuidelineXmlFileReader.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Xml.Linq;
+
+namespace GuidelineXmlToMD
+{
+    static class GuidelineXmlFileReader
+    {
+        public const string _Guideline = "guideline";
+        public const string _Key = "key";
+        public const string _Severity = "severity";
+
+        public const string _Section = "section";
+        public const string _Subsection = "subsection";
+        public const string _Comments = "comments";
+
+
+        static public ICollection<Guideline> ReadExisitingGuidelinesFile(string pathToExistingGuidelinesXml)
```

Comment:
nit: disjoint of ordering of parameters.
```suggestion
        public static ICollection<Guideline> ReadExisitingGuidelinesFile(string pathToExistingGuidelinesXml)
```